packages/react/src/stories/ic-text-field.stories.mdx

```diff
@@ -482,3 +482,207 @@ export const Uncontrolled = () => {
     ></IcTextField>
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  autocapitalize: "off",
+  autocomplete: "off",
+  autocorrect: "off",
+  autoFocus: false,
+  disabled: false,
+  fullWidth: false,
+  helperText: "",
+  hideLabel: false,
+  inputId: "ic-text-field-input-0",
+  inputmode: "text",
+  label: "Text Field",
+  maxCharacters: 0,
+  maxLength: 0,
+  minCharacters: 0,
+  name: "0",
+  placeholder: "",
+  readonly: false,
+  required: false,
+  resize: false,
+  rows: 1,
+  size: "default",
+  spellcheck: false,
+  type: "text",
+  validationInline: false,
+  validationStatus: "",
+  validationText: "",
+  debounce: 0
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    argTypes={{
+      autocapitalize: {
+        options: ["off", "none", "on", "sentences", "words", "characters"],
+        control: { type: "select" }
+      },
+      autocomplete: {
+        options: [
+          "on",
+          "off",
+          "name",
+          "honorific-prefix",
+          "given-name",
+          "additional-name",
+          "family-name",
+          "honorific-suffix",
+          "nickname",
+          "email",
+          "username",
+          "new-password",
+          "current-password",
+          "one-time-code",
+          "organization-title",
+          "organization",
+          "street-address",
+          "address-line1",
+          "address-line2",
+          "address-line3",
+          "address-level4",
+          "address-level3",
+          "address-level2",
+          "address-level1",
+          "country",
+          "country-name",
+          "postal-code",
+          "cc-name",
+          "cc-given-name",
+          "cc-additional-name",
+          "cc-family-name",
+          "cc-family-name",
+          "cc-number",
+          "cc-exp",
+          "cc-exp-month",
+          "cc-exp-year",
+          "cc-csc",
+          "cc-type",
+          "transaction-currency",
+          "transaction-amount",
+          "language",
+          "bday",
+          "bday-day",
+          "bday-month",
+          "bday-year",
+          "sex",
+          "tel",
+          "tel-country-code",
+          "tel-national",
+          "tel-area-code",
+          "tel-local",
+          "tel-extension",
+          "impp",
+          "url",
+          "photo"
+        ],
+        control: { type: "select" }
+      },
+      autocorrect: {
+        options: ["off", "on"],
```

Comment:
Not sure if I've misunderstood what this prop should do, it's not working as I expect based off the prop description: _"The state of autocorrection the browser can apply when the user is entering/editing the text value."_
<img width="688" alt="Screenshot 2024-06-19 at 13 52 25" src="https://github.com/mi6/ic-ui-kit/assets/113986285/e67dbbe4-ca89-48ca-9b06-5f6c583dce6d">